solutions_python_book/lazy_student/lazy_student.py

```diff
@@ -0,0 +1,85 @@
+import sys
+
+
+def main():
+    file_home_work_name = "home_work.txt"
+    file_solution_name = "solutions.txt"
+    # file_home_work_name = sys.argv[1]
+    # file_solution_name = sys.argv[2]
+
+    try:
+        with open(file_home_work_name, "r") as f_read:
+            all_exercises = f_read.readlines()
+
+            with open(file_solution_name, "w") as f_write:
```

Comment:
You already read `file_home_work_name`.
you can ident this block to the left, you don't need f_read anymore

Reply:
in general, don't reach high nesting level